website/content/docs/upgrading/version-guides/0.11.0.mdx

```diff
@@ -0,0 +1,92 @@
+---
+layout: docs
+page_title: Version 0.11.0 - Upgrade Guide
+description: |-
+This upgrade guide describes any considerations for upgrading to Waypoint 0.11.0.
+---
+
+# Upgrading to Waypoint 0.11.0 from 0.10.x
+
+Waypoint 0.10.x can be upgraded to Waypoint 0.11.0 using a [standard upgrade](/waypoint/docs/upgrading#standard-upgrade).
+There are some considerations depending on what platform your server is installed on.
+There are no breaking changes in the protocol or `waypoint.hcl` configuration file.
+Due to the changes in `runner` management, we cannot guarantee that all Waypoint
+components (CLI, entrypoint, and server) are forwards and backwards
+compatible. We encourage users to have all components on the same version.
+
+## Server Snapshot
+
+Waypoint servers at versions 0.10.4 and 0.10.5 have a bug with taking snapshots,
```

Comment:
Just a thought, could be neat to link to the bug fix PR here! Not required though.
```suggestion
Waypoint servers at versions 0.10.4 and 0.10.5 have a [bug with taking snapshots](https://github.com/hashicorp/waypoint/pull/4523),
```

Reply:
I agree!